### PR TITLE
Fixes #14018: Update rudder reports to match modern packaging recommendation

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14018

### CODE REVIEW

File: rudder-reports/SOURCES/Makefile [hunk 1]
```diff
@@ -16,23 +16,59 @@
 #
 #####################################################################################
 
-.DEFAULT_GOAL := localdepends
+.DEFAULT_GOAL := build
 
-RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
+#RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
+RUDDER_VERSION_TO_PACKAGE = 5.0.1
```
Comment (amousset) [inline rudder-reports/SOURCES/Makefile]: test leftover here

**Comment by peckpeck** [comment]
Commit modified

File: rudder-reports/debian/rules [hunk 2]
```diff
@@ -1,65 +1,30 @@
...
 # -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# This file was originally written by Joey Hess and Craig Small.
-# As a special exception, when this file is copied by dh-make into a
-# dh-make output file, you may use that output file without restriction.
-# This special exception was added by Craig Small in version 0.37 of dh-make.
 
 # Uncomment this to turn on verbose mode.
-#export DH_VERBOSE=1
+export DH_VERBOSE=1
```
Comment (amousset) [inline rudder-reports/debian/rules]: It this on purpose?

Comment (peckpeck) [reply rudder-reports/debian/rules]: yes, all our build are verbose by default, to have maximum logs available for debug

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**